.github/workflows/ci.yml

```diff
@@ -100,12 +102,16 @@ jobs:
           file /tmp/x86_64-unknown-linux-gnu/diesel_ext
           file /tmp/x86_64-unknown-linux-gnu/cargo-audit
           file /tmp/x86_64-unknown-linux-gnu/cargo-watch
+          file /tmp/x86_64-unknown-linux-gnu/cargo-dylint
+          file /tmp/x86_64-unknown-linux-gnu/dylint-link
           file /tmp/x86_64-unknown-linux-gnu/penguin
           /tmp/x86_64-unknown-linux-gnu/sccache --help
           /tmp/x86_64-unknown-linux-gnu/diesel --help
           /tmp/x86_64-unknown-linux-gnu/diesel_ext --help
           /tmp/x86_64-unknown-linux-gnu/cargo-audit --help
           /tmp/x86_64-unknown-linux-gnu/cargo-watch --help
+          RUSTUP_TOOLCHAIN="stable" /tmp/x86_64-unknown-linux-gnu/cargo-dylint --help
```

Comment:
this is on purpose - dylint tools relies on `rustup` (even for --help, I made a ticket in GH for that), and since we don't have `rustup` in the worker (we do have inside of a docker), we can fake it by setting this env variable.

Reply:
https://github.com/trailofbits/dylint/issues/1172